data/scopes/defaults.ini

```diff
@@ -2,10 +2,9 @@
 inheritFrom = ""
 scopeType = "defaults"
 displayName = ""
-last_edit_time = 1571296800
 
 [data]
-fanvil_model_mac_template = "fanvil-MAC_ADDRESS.tmpl"
+tmpl_fanvil_phone = "fanvil-x3.tmpl"
```

Comment:
why have you renamed variable from "fanvil_model_mac_template" to "tmpl_fanvil_phone"? Watch out: its name is still fanvil_model_mac_template in other scope files. 
Refactor is ok for me, but I'd prefear to refactor all variable names (also for other brands) at the end to maintain uniformity